Remove THC ScalarConvert

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #65492
* #65491
* #65472
* __->__ #65471
* #65470

Differential Revision: [D31148182](https://our.internmc.facebook.com/intern/diff/D31148182)

peterbell10 committed:
Remove THC ScalarConvert

[ghstack-poisoned]

peterbell10 committed:
Update on "Remove THC ScalarConvert"




[ghstack-poisoned]

peterbell10 committed:
Update on "Remove THC ScalarConvert"




[ghstack-poisoned]

peterbell10 committed:
Update on "Remove THC ScalarConvert"




[ghstack-poisoned]

peterbell10 committed:
Update on "Remove THC ScalarConvert"




[ghstack-poisoned]

ngimel commented:
@ngimel has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148182).

peterbell10 committed:
Remove manual hipifying include path on "Remove THC ScalarConvert"


Differential Revision: [D31148182](https://our.internmc.facebook.com/intern/diff/D31148182)

[ghstack-poisoned]

ngimel commented:
@ngimel has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148182).
